src/logs/sentry.rs

```diff
@@ -161,8 +163,8 @@ impl Stat for SuccessfulScan {
         }
     }
 
-    fn report(&self) {
-        println!("Total successful scans: {}", self.total);
+    fn report(&self, formatter: &mut std::fmt::Formatter) -> fmt::Result {
+        formatter.write_fmt(format_args!("Total successful scans: {}", self.total))
```

Comment:
Why not the `write/writeln` macro?

Reply:
Just used to do it like that, but totally true.